apps/web/screens/UniversitySearch/LandingPage.tsx

```diff
@@ -0,0 +1,220 @@
+              id="searchuniversity"
+              name="filterInput"
+              size="md"
+              value={searchTerm}
+              className={cn(styles.searchInput)}
+              backgroundColor="blue"
+              onChange={(e) => {
+                setSearchTerm(e.target.value)
+              }}
+              onKeyDown={(k) => {
+                if (k.code === 'Enter') {
+                  routeToSearch()
+                }
+              }}
+            />
+            <button
+              className={cn(styles.searchIcon)}
+              onClick={() => routeToSearch()}
+            >
+              <Icon size="large" icon="search" color="blue400" />
+            </button>
+          </GridColumn>
+        </Box>
+      }
+    >
+      {organizationPage?.bottomSlices.map((slice) => (
+        <SliceMachine
+          key={slice.id}
+          slice={slice}
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore make web strict
+          namespace={namespace}
+          slug={organizationPage.slug}
+          fullWidth={true}
+          params={{
+            latestNewsSliceBackground:
+              organizationPage.theme === 'landing_page' ? 'white' : 'purple100',
+            latestNewsSliceColorVariant:
+              organizationPage.theme === 'landing_page' ? 'blue' : 'default',
+          }}
+        />
+      ))}
+      {organizationPage?.theme === 'landing_page' && (
+        <LandingPageFooter
+          footerItems={organizationPage.organization?.footerItems}
+        />
+      )}
+    </OrganizationWrapper>
+  )
+}
+
+LandingPage.getProps = async ({ apolloClient, locale }) => {
+  const [
+    {
+      data: { getOrganizationPage },
+    },
+    {
+      data: { getOrganization },
+    },
+    namespace,
+  ] = await Promise.all([
+    apolloClient.query<Query, QueryGetOrganizationPageArgs>({
+      query: GET_ORGANIZATION_PAGE_QUERY,
+      variables: {
+        input: {
+          slug: locale === 'is' ? 'haskolanam' : 'university-studies',
+          lang: locale as ContentLanguage,
+        },
+      },
+    }),
+    apolloClient.query<Query, QueryGetOrganizationPageArgs>({
+      query: GET_ORGANIZATION_QUERY,
+      variables: {
+        input: {
+          slug: locale === 'is' ? 'haskolanam' : 'university-studies',
+          lang: locale as ContentLanguage,
+        },
+      },
+    }),
+    apolloClient
+      .query<Query, QueryGetNamespaceArgs>({
+        query: GET_NAMESPACE_QUERY,
+        variables: {
+          input: {
+            namespace: 'universityGateway',
+            lang: locale,
+          },
+        },
+      })
+      .then((variables) =>
+        variables?.data?.getNamespace?.fields
+          ? JSON.parse(variables.data.getNamespace.fields)
+          : {},
+      ),
+  ])
+
+  if (!getOrganizationPage && !getOrganization?.hasALandingPage) {
+    throw new CustomNextError(404, 'Organization page not found')
+  }
```

Comment:
And here the 404 is in english.
Are these perhaps routed in a way I'm not familiar with?

Reply:
They should all be the same error but I can't have them translated since this is server code